recipe/meta.yaml

```diff
@@ -1,13 +1,13 @@
 {% set name = "arxiv-collector" %}
-{% set version = "0.1.1" %}
+{% set version = "0.2.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
```

Comment:
If you put up the tarball on github and point this there it should work, I think.